src/vivarium_public_health/results/disease.py

```diff
@@ -181,26 +168,43 @@ def on_time_step_prepare(self, event: Event) -> None:
     def aggregate_state_person_time(self, x: pd.DataFrame) -> float:
         return len(x) * to_years(self.step_size())
 
-    ##################
-    # Report methods #
-    ##################
-
-    def formatter(
-        self,
-        measure_name: str,
-        entity_type: str,
-        entity: str,
-        sub_entity_col: str,
-        measure: str,
-        results: pd.DataFrame,
-    ) -> pd.DataFrame:
+    ##############################
+    # Results formatting methods #
+    ##############################
+
+    def format(self, measure, results) -> pd.DataFrame:
         results = results.reset_index()
-        # Remove no_transitions
-        if measure_name == "transition_count":
-            results = results[results[sub_entity_col] != "no_transition"]
-        results.rename(columns={sub_entity_col: COLUMNS.SUB_ENTITY}, inplace=True)
-        results[COLUMNS.MEASURE] = measure_name
-        results[COLUMNS.ENTITY_TYPE] = entity_type
-        results[COLUMNS.ENTITY] = entity
-
-        return results[[c for c in results.columns if c != COLUMNS.VALUE] + [COLUMNS.VALUE]]
+        if "transition_count_" in measure:
```

Comment:
Why is this check including the trailing underscore?

Reply:
Because all the names are `transition_count_{self.risk}`